src/pages/LogIn/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react";
+import "./login.css";
+import { Form } from "../../components";
+import { useDispatch } from "react-redux";
+import { fetchUserData } from "../../state/LogIn/loginSlice";
+import { useNavigate } from "react-router-dom";
+const logo = "/assets/images/logo.png";
+const leftImage = "/assets/images/leftImage.png";
+const rightImage = "/assets/images/rightImage.png";
+
+const LogIn = () => {
+  const dispatch = useDispatch();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const navigate = useNavigate();
+  const handleSignup = () => {
+    navigate("/signup");
+  };
+
+  const handleFormSubmit = (e) => {
+    e.preventDefault();
+    console.log(email, password);
+    dispatch(
+      fetchUserData({
+        email: email,
+        password: password,
+      })
+    );
+  };
+  return (
+    <div className="h-screen bg-cyan-50">
+      <div className="leftImage">
+        <img src={leftImage} alt="background" />
+      </div>
+
+      <div className="logo-bg">
+        <img
+          className={`logo h-32 w-48 bg-blue-800 object-contain p-3`}
+          src={logo}
+          alt=""
+        />
+      </div>
+      <div>
+        <div className=" border-primaryBorder m-auto w-full max-w-md rounded-lg border bg-white py-10 px-16 shadow-xl">
+          <h1 className="text-bold text-primary mt-4 mb-12 text-center text-2xl font-medium">
+            Log in to your account
+          </h1>
+
+          <form>
+            <Form
+              type="email"
+              value={email}
+              name="email"
+              placeholder="Enter email"
+              onChange={(e) => setEmail(e.target.value)}
+            />
+
+            <Form
+              type="password"
+              value={password}
+              name="password"
+              placeholder="Enter Password"
+              onChange={(e) => setPassword(e.target.value)}
+            />
+
+            <div className="mt-6 ">
+              <button
+                onClick={handleFormSubmit}
+                style={{ width: "95%" }}
```

Comment:
Why inline?

Reply:
In form component, input field have a fixed width. I have to put same width in button. Can't fix this width with tailwind.

Reply:
done